### DISCUSSION TITLE
Physiologically-Based Pharmacokinetic (PBPK) Modeling of Fentanyl in Adults and Pediatric Patients

### DISCUSSION THREAD

**Discussion by LkKov** [Open-Systems-Pharmacology/Forum#735] [Announcement]
Dear OSP-community,

We have recently been working on PBPK modeling of fentanyl in adults and pediatrics. First, an adult PBPK model has been thoroughly developed, which is able to predict fentanyl and norfentanyl venous and arterial blood plasma concentration-time profiles after intravenous administration. The adult PBPK model has been used for the assessment of the fentanyl drug-drug interaction with voriconazole.
Subsequently, the adult PBPK model has been scaled to several pediatric populations and was used to predict plasma concentration-time profiles as well as individual and mean clearances. Furthermore, the pediatric PBPK model was used to simulate differences in peak plasma concentrations after bolus injections and short infusions.

The corresponding manuscript, including comprehensive documentation, has been published online in Pharmaceutics. The PK-Sim model files presenting the PBPK models including simulations of clinical studies can be found [here](https://github.com/Open-Systems-Pharmacology/Fentanyl-Model/).

Related publication:
Kovar L., Weber A., Zemlin M., Kohl Y., Bals R., Meibohm B., Selzer D., Lehr T. Physiologically-Based Pharmacokinetic (PBPK) Modeling Providing Insights into Fentanyl Pharmacokinetics in Adults and Pediatric Patients. Pharmaceutics 2020, 12, 908. https://doi.org/10.3390/pharmaceutics12100908

Best regards,
Lukas

**Comment by TWendl** [comment]
Congratulations!
